### DISCUSSION TITLE
CkEditor 42 new builder not compatible with custom build

### DISCUSSION THREAD

**Discussion by adambalint-srg** [Laravel-Backpack/community-forum#1068] [Q&A (Help)] [answered]
CkEditor new 42 version has a new builder (https://ckeditor.com/ckeditor-5/builder/). But it doesn't build a single file which is comaptible with backpack's custom build option, but a module-js plus a main.js file, and I can't use it here. What is the official way to use custom builds with this new ckeditor version?

**Comment by pxpm** [comment]
Hello @adambalint-srg 

Thanks for the question. Did our docs didn't help here? https://backpackforlaravel.com/docs/6.x/crud-fields#ckeditor-custom-builds

Let me know what you've tried and what didn't work as you expected so that I can reproduce it on my side. 

Cheers

**Comment by adambalint-srg** [comment]
@pxpm 
Of course I tried to make it based on your description, and in a previous project I could do it. But if you go through the new CkEditor builder: (https://ckeditor.com/ckeditor-5/builder/) it won't give you a single built file, which can be included in backpack.

**Comment by pxpm** [comment]
It seems their build process was updated and we need to update our docs/code. Thanks for the heads up 🙏 

I will move this discussion into an issue in backpack/crud repository, you can follow up there and subscribe to the issue to get notified when we work on it 👍 

Cheers

**Reply by adambalint-srg** [reply]
Do you have any update on this? My client need this functionality, and we're getting close to 2 months. We can't really use this functionality of the whole package, and this is problematic.

**Reply by pxpm** [reply]
Hello @adambalint-srg sorry for the delay, but I don't have an answer for you yet. 

To give you some more context, we are working closely with CKeditor developers to solve a Licensing issue. CKeditor dropped the "free" version, so we need to come with an agreement with them to include CKeditor in our software. 
Until that issue is settled we can't be spending time on this as we don't know how things will turn out. Unfortunately I don't have a date for it to settle, emails have been going back and forth during last months, but we haven't reached an agreement yet. 

I'd recommend you use other wysiwyg editor (summernote for example). Or alternatively publish the field we currently have https://backpackforlaravel.com/docs/6.x/crud-how-to#how-to-publish-a-column-field-filter-button-and-modify-it and do the needed modifications. As soon as we have an answer to this issue you will get notified. 

Cheers

**Reply by tabacitu** [reply, marked as answer]
Agree with Pedro here. Heads-up, this has also been reported here - https://github.com/Laravel-Backpack/docs/issues/593 - and there are steps there to craft a solution for yourself.
